<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonPut">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip_server_input">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ip_server_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ip_server_disconnect">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ip_server_connect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QSlider" name="min_slider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="min_display">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="min_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="max_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QSlider" name="max_slider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="max_display">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Timing(s)</string>
    </property>
   </widget>
   <widget class="QSlider" name="time_slider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="time_display">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="server_status">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>256</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_server_input">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="port_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>port</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>min_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>min_display</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>max_display</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>time_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>time_display</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
